frontend/src/router/index.ts

```diff
@@ -67,6 +67,10 @@ const router = createRouter({
 });
 
 router.beforeEach((to) => {
+  // Removed for now
+  // const navStore = useNavStore();
```

Comment:
Same here - wondering if we need to keep the commented code?

Reply:
For this one, we want to revisit breadcrumbs at a later date, so this should be preserved.